ccdImports/configFiles/CCD_Probate_Legacy_Search/AuthorisationCaseField.json

```diff
@@ -16,5 +16,15 @@
   {"CaseTypeID": "LegacySearch", "CaseFieldID": "deceasedDateOfBirth", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
   {"CaseTypeID": "LegacySearch", "CaseFieldID": "deceasedDateOfDeath", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
   {"CaseTypeID": "LegacySearch", "CaseFieldID": "legacySearchResultRows", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
-  {"CaseTypeID": "LegacySearch", "CaseFieldID": "doNotSaveCase", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"}
+  {"CaseTypeID": "LegacySearch", "CaseFieldID": "doNotSaveCase", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
+  {"CaseTypeID": "LegacySearch", "CaseFieldID": "customHistoryViewer", "UserRole": "caseworker-probate-superuser", "CRUD": "R"},
```

Comment:
some of these are already setup on the authcasefield for superuser, eg. deceasedSurname